src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -217,7 +217,7 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 	 * Enables opening the contextual actions, if any, that are available
 	 * and registering of command finished listeners
 	 */
-	get contextualActions(): IContextualAction | undefined { return this._contextualActionAddon; }
+	get quickFixActions(): QuickFixAction | undefined { return this._quickFixAddon; }
```

Comment:
this plural vs singular is bothering me... maybe should rename `QuickFixAction` to `QuickFixActions` or `QuickFixHandler`?

Reply:
Just `quickFix` works fine, that aligns with `statusList`